sdk/ai/Azure.AI.Inference/samples/Sample8_ChatCompletionsWithTelemetry.md

```diff
@@ -0,0 +1,127 @@
+var appInsightsConn = System.Environment.GetEnvironmentVariable("APP_INSIGHTS_CONNECTION_STR");
+```
+
+In this example we will take these values from the environment variables. To get the telemetry, we will need to listen to [ActivitySource](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/Diagnostics.md#enriching-generic-http-activities-with-azure-request-identifiers) named `Azure.AI.Inference.ChatCompletionsClient`. To allow telemetry collection we will create the listeners `tracerProvider`, which will listen to activity and `meterProvider`, listening to meter. We will add console exporters to both providers with the line `AddConsoleExporter()` and monitor exporter for Application Insights export: `AddAzureMonitorMetricExporter`. 
+
+```C# Snippet:Azure_AI_Inference_TelemetrySyncScenario_providers
+const string ACTIVITY = "Azure.AI.Inference.ChatCompletionsClient";
+using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+    .AddSource(ACTIVITY)
+    .ConfigureResource(r => r.AddService("MyServiceName"))
+    .AddConsoleExporter()
+    .AddAzureMonitorTraceExporter(options =>
+    {
+        options.ConnectionString = appInsightsConn;
+    })
+    .AddOtlpExporter()
+    .Build();
+
+using var meterProvider = Sdk.CreateMeterProviderBuilder()
+    .AddMeter(ACTIVITY)
+    .ConfigureResource(r => r.AddService("MyServiceName"))
+    .AddConsoleExporter()
+    .AddAzureMonitorMetricExporter(options =>
+    {
+        options.ConnectionString = appInsightsConn;
+    })
+    .AddOtlpExporter()
+    .Build();
+```
+
+To export only events, we need to define `tracerProvider`, to export also metrics, we need to we also need to define `meterProvider`. Metrics cannot be exported without events.
+After we have initialized the providers, we will use simple completions example from [Sample1](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/ai/Azure.AI.Inference/samples/Sample1_ChatCompletions.md).
+```C# Snippet:Azure_AI_Inference_TelemetrySyncScenario_inference
+var client = new ChatCompletionsClient(
+    endpoint,
+    credential,
+    new ChatCompletionsClientOptions());
+
+var requestOptions = new ChatCompletionsOptions()
+{
+    Messages =
+    {
+        new ChatRequestSystemMessage("You are a helpful assistant."),
+        new ChatRequestUserMessage("What is the capital of France?"),
+    },
+    Model = model,
+    Temperature = 1,
+    MaxTokens = 1000
+};
+// Call the enpoint and output the response.
+Response<ChatCompletions> response = client.Complete(requestOptions);
+System.Console.WriteLine(response.Value.Choices[0].Message.Content);
+```
+
+## Running the application
+**IMPORTANT!** To switch on the telemetry we need to set environment variable `OPENAI_EXPERIMENTAL_ENABLE_OPEN_TELEMETRY` to "1" or "true". To enable the collection of chat requests and responses please set `AZURE_TRACING_GEN_AI_CONTENT_RECORDING_ENABLED` variable to "1". It can be done in IDE or in the console as outlined below.
```

Comment:
OPENAI_EXPERIMENTAL_ENABLE_OPEN_TELEMETRY, what is the reason this name contains "EXPERIMENTAL"?